docker/mxnet-model-server.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+MMS_DIR='/mxnet-model-server'
+CONFIG_FILE=$MMS_DIR'/mms_app.conf'
+GUNICORN_ARGS='gunicorn'
+MXNET_ARGS='mxnet'
+NGINX_ARGS='nginx'
+MMS_ARG='mms'
+NGINX_CONFIG_FILE='/etc/nginx/conf.d/virtual.conf'
+MMS_CONFIG_FILE=''
+
+# Start MxNet model server. This function expects one argument which is the application config file.
+start_mms() 
+{
+    gunicorn_arguments=''
+    # TODO: Check if this process is already running . If yes ignore start and throw warning.
```

Comment:
Can you file an issue for this and link it here? it will help us track it.

Reply:
Please file an issue AND paste the link in the code, which makes it easier to trace.

Reply:
Added the link to the issue here.